### PR TITLE
Add testcase for 3122

### PR DESCRIPTION
**Pull request by maryvictol**
This PR adds testcases for Run jobs in other regions in case of insufficient capacity https://github.com/epam/cloud-pipeline/issues/3122

### CODE REVIEW

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_1.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- Admin user
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Open the **Settings** page
+3. Click the **CLOUD REGIONS** tab
+4. Select `region1` on the left panel
+5. Save the ***Run shift policy*** checkbox state, then tick it if needed
+6. Repeat steps 4-5 for `region2`
+
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_1.md:21]: As there can be more than 2 regions for the selected Cloud Provider, their shift policies shall be disabled for the testing period.
Therefore, I suggest to add steps like the following:
7. Select `region3` on the left panel if it exists
8. Save the ***Run shift policy*** checkbox state, then untick it if needed
9. Repeat steps 7-8 for remaining regions of the selected Cloud Provider (if they exist)

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_1.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the Tools page | |
+| 2 | Select test tool | |
+| 3 | At the tool page, hover over the **Run v** button | |
+| 4 | Click the **Custom settings** button in the list | |
+| 5 | Expand the **Exec environment** section | |
+| 6 | Select `p2.8xlarge` value for **Node type** | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_1.md]: I suggest not to specify node type explicitly here. And change the step to something like:
"Select the **Node type** from the prerequisites"

And specify in the Prerequisites:
"- Node type `type1` of rare compute instance type for the current Cloud Provider which launch attempt can lead to the exception of insufficient capacity of such node types for the Cloud Region"

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_2.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as admin user from the prerequisites | |
+| 2 | Open the Tools page | |
+| 3 | Select test tool | |
+| 4 | At the tool page, hover over the **Run v** button | |
+| 5 | Click the **Custom settings** button in the list | |
+| 6 | Expand the **Exec environment** section | |
+| 7 | Select `p2.8xlarge` value for **Node type** | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_2.md]: I suggest not to specify node type explicitly here. And change the step to something like:
"Select the **Node type** from the prerequisites"

And specify in the Prerequisites:
"- Node type `type1` of rare compute instance type for the current Cloud Provider which launch attempt can lead to the exception of insufficient capacity of such node types for the Cloud Region"

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_2.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+| 8 | Select `region1` value for **Cloud Region** | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_2.md]: Change to:
"Select `region1` value (same as in step 4 of the preparations of case [3122_1](3122_1.md)) for **Cloud Region**"

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_3.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as admin user from the prerequisites | |
+| 2 | Open the Tools page | |
+| 3 | Select test tool | |
+| 4 | At the tool page, hover over the **Run v** button | |
+| 5 | Click the **Custom settings** button in the list | |
+| 6 | Expand the **Exec environment** section | |
+| 7 | Select `p2.8xlarge` value for **Node type** | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_3.md]: I suggest not to specify node type explicitly here. And change the step to something like:
"Select the **Node type** from the prerequisites"

And specify in the Prerequisites:
"- Node type `type1` of rare compute instance type for the current Cloud Provider which launch attempt can lead to the exception of insufficient capacity of such node types for the Cloud Region"

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+5. Save the ***Run shift policy*** checkbox state, then untick it if needed
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Open the Tools page | |
+| 2 | Select test tool | |
+| 3 | At the tool page, hover over the **Run v** button | |
+| 4 | Click the **Custom settings** button in the list | |
+| 5 | Expand the **Exec environment** section | |
+| 6 | Select `p2.8xlarge` value for **Node type** | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md]: I suggest not to specify node type explicitly here. And change the step to something like:
"Select the **Node type** from the prerequisites"

And specify in the Prerequisites:
"- Node type `type1` of rare compute instance type for the current Cloud Provider which launch attempt can lead to the exception of insufficient capacity of such node types for the Cloud Region"

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_3.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+| 8 | Select `region1` value for **Cloud Region** | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_3.md]: Change to:
"Select `region1` value (same as in step 4 of the preparations of case [3122_1](3122_1.md)) for **Cloud Region**"

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+**Prerequisites**:
+- Admin user
+- Perform [3122_1](3122_1.md) case
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Open the **Settings** page
+3. Click the **CLOUD REGIONS** tab
+4. Select `region1` on the left panel
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md]: Change to:
"Select `region1` (same as in step 4 of the preparations of case [3122_1](3122_1.md)) on the left panel"

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+5. Save the ***Run shift policy*** checkbox state, then untick it if needed
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md]: Change to:
"Untick the ***Run shift policy*** checkbox"

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+| 4 | Click the **Custom settings** button in the list | |
+| 5 | Expand the **Exec environment** section | |
+| 6 | Select `p2.8xlarge` value for **Node type** | |
+| 7 | Select `region1` value for **Cloud Region** | |
+| 8 | Expand the **Advanced** section | |
+| 9 | Select `On-demand` value for **Price type** | |
+| 10 | Launch the run | |
+| 11 | At the Runs page, click the just-launched run | |
+| 12 | Wait until ***RestartPipelineRun*** task appears | <li> Run is stopped <li> Message `Server failure reason: Insufficient instance capacity.` appears under the run title |
+| 13 | Click ***RestartPipelineRun*** task | Records `Run '<runID>' failed to start. <br> Region relaunch is not available for runs for region '<region1>'.` are shown in the log |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md]: Add below:

After the test, open the **CLOUD REGIONS** tab of the **Settings** page and return states to the ***Run shift policy*** checkboxes for all regions saved at steps 4-9 of the preparations of case [3122_1](3122_1.md)

**Review by NShaforostov — APPROVED** [review]
Fix the notes before the merge, please. See comments for details.

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md (hunk 10 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md (hunk 9 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md (hunk 8 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_3.md (hunk 7 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_3.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_2.md (hunk 4 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_2.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md (hunk 6 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_4.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_3.md (hunk 5 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_3.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_2.md (hunk 3 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_2.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_1.md (hunk 2 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_1.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_1.md (hunk 1 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3122_1.md:21]: Done

**Pull request merged**